pkg/registry/rbac/clusterrolebinding/registry.go

```diff
@@ -61,8 +60,8 @@ type AuthorizerAdapter struct {
 	Registry Registry
 }
 
-func (a AuthorizerAdapter) ListClusterRoleBindings() ([]*rbacv1.ClusterRoleBinding, error) {
-	list, err := a.Registry.ListClusterRoleBindings(genericapirequest.NewContext(), &metainternalversion.ListOptions{})
+func (a AuthorizerAdapter) ListClusterRoleBindings(ctx context.Context) ([]*rbacv1.ClusterRoleBinding, error) {
+	list, err := a.Registry.ListClusterRoleBindings(ctx, &metainternalversion.ListOptions{})
```

Comment:
will this break if the context came from a request that was namespace-scoped?

Reply:
I don't think it will, but I lack understanding in how Listers works under the hood with Namespaced Context. Would a helper like mentioned in comment above help here?